src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+
+  if (days < 3) {
+    count = days * 40;
+  }
+
+  if (days >= 3) {
+    count = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    count = days * 40 - 50;
+  }
+
+  return count;
```

Comment:
Actually, you can save `days * 40` as a variable and just immediately return the full price inside `if`s